packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -113,7 +127,12 @@ export class JobService {
       .div(100);
     const totalAmount = BigNumber.from(fundAmountInWei).add(totalFee);
```

Comment:
The totalAmount is the fundAmount, fee should be deducted from the fund amount. The only fee can be added to the fund amount is the job launcher fee, because it is not paid through the escrow contract